rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```diff
@@ -38,10 +38,12 @@ applications are often leveraged by adversaries to execute code and evade detect
 """
 from = "now-9m"
 index = [
-    "logs-endpoint.events.process-*",
+    "endgame-*",
     "logs-endpoint.events.network-*",
-    "winlogbeat-*",
+    "logs-sentinel_one_cloud_funnel.*",
+    "logs-crowdstrike.fdr*",
```

Comment:
```suggestion
```
CS is inconsistent with process.entity_id for now, network events don't have it yet (this is documented and pending fix), and it don't populate process.parent.executable